### ISSUE TITLE
CloudFlare is stripping UTM tags again

### ISSUE THREAD

**Issue by TheoChevalier** [mozilla/foundation.mozilla.org#7329]
Looks like what we had in place is not working anymore, UTM tags are stipped on prod during redirects (works on staging)

Test URL https://foundation.mozilla.org/campaigns/apples-anti-tracking-plans-iphone/?subscribed=1&utm_source=email&utm_medium=email&utm_campaign=2020advocacy-fr&utm_content=appleidfathanks&utm_term=5382510

@tomusher Could you check the custom code you wrote for the worker is still in place?

**Comment by benhohner** [comment]
@TheoChevalier Upon further investigation looks like a localization regression that only works with links without the locale.  https://github.com/mozilla/foundation.mozilla.org/issues/7329 @tomusher has checked and it's not Cloudflare this time

**Comment by Pomax** [comment]
@ben if that were the case, we'd expect to see it fail on staging, too, but it only happens on production. This link works fine:

https://foundation.mofostaging.net/campaigns/apples-anti-tracking-plans-iphone/?subscribed=1&utm_source=email&utm_medium=email&utm_campaign=2020advocacy-fr&utm_content=appleidfathanks&utm_term=5382510

This link does not:

https://foundation.mozilla.org/campaigns/apples-anti-tracking-plans-iphone/?subscribed=1&utm_source=email&utm_medium=email&utm_campaign=2020advocacy-fr&utm_content=appleidfathanks&utm_term=5382510

**Comment by benhohner** [comment]
@Pomax true! Reverting. @tomusher can you take a look at this again?

**Comment by benhohner** [comment]
@tomusher @TheoChevalier Update: seems to only happen if you're logged in to Wagtail. Otherwise, it works properly

**Comment by TheoChevalier** [comment]
Oooh nice! Thanks for figuring this out!

**Comment by tomusher** [comment]
Good catch thank you @benhohner - turns out this was the worker.

It was configured to strip querystrings from every request, check if the request was cacheable (i.e. user isn't logged in), and then only reapplies the stripped values if the request *was* cacheable.

I've moved the querystring stripping logic so it only happens *after* we've determined whether the request is cacheable which seems to have resolved this.
